neurom/features/morphology.py

```diff
@@ -684,3 +684,41 @@ def shape_factor(morph, neurite_type=NeuriteType.all, projection_plane="xy"):
     """
     projected_points = _unique_projected_points(morph, projection_plane, neurite_type)
     return np.nan if len(projected_points) == 0 else morphmath.shape_factor(projected_points)
+
+
+@feature(shape=())
+def length_fraction_above_soma(morph, neurite_type=NeuriteType.all, up="Y"):
+    """Returns the length fraction of the segments that have their midpoints higher than the soma.
```

Comment:
What is the expected accuracy of this feature? Is it relevant to split the segments that cross the 0 of the considered axis to improve accuracy?

Reply:
Ok! :-)